test/flipper_app/authorize_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../test_helper'
+require 'mocha/setup'
+require 'flipper_app/authorize'
+
+class AssignmentsApiTest < Minitest::Test
+  def test_unauthenticated_users_rejected
+    User.stubs(:where).returns(authorized_users)
```

Comment:
That is a common problem I hit. I haven't typically thought to stub my own methods because of POODR's warning against this practice, but with a method this small, I absolutely agree that the effect is equivalent with a method this small, and it's much more resilient to change. Not to mention that it side-steps the annoying side effects I was having when stubbing this AR method with that led to the duplication. I appreciate you having me think about this! 🎉 

Fixed in f41f595.